alea/utils.py

```diff
@@ -81,6 +83,18 @@ def _package_path(sub_directory):
     return pkg_resources.resource_filename('alea', f'{sub_directory}')
 
 
+def formatted_to_asterisked(formatted):
+    """
+    Convert formatted string to asterisk
+    Sometimes a parameter(usually shape parameter) is not specified in formatted string,
+    this function replace the parameter with asterisk.
+    """
+    asterisked = formatted
+    for found in re.findall("\{(.*?)\}", formatted):
```

Comment:
This idea is borrowed from https://stackoverflow.com/questions/27881366/regular-expressions-and.

Reply:
Awesome, that's a very clean solution! 😊